tests/unit/test_model_setup.py

```diff
@@ -0,0 +1,918 @@
+        np.testing.assert_equal(stiff.lmbda, np.array([1.0]))
+        np.testing.assert_equal(
+            stiff.values, pp.FourthOrderTensor(np.array([1.0]), np.array([1.0])).values
+        )
+        # Source term
+        source_mech = data[pp.PARAMETERS][mech_kw]["source"]
+        np.testing.assert_equal(source_mech, np.array([0.0, 0.0]))
+        # Biot alpha
+        biot_alpha_mech = data[pp.PARAMETERS][mech_kw]["biot_alpha"]
+        np.testing.assert_equal(biot_alpha_mech, np.array([1.0]))
+        # Reference pressure
+        p_reference = data[pp.PARAMETERS][mech_kw]["p_reference"]
+        np.testing.assert_equal(p_reference, np.array([0.0]))
+        # Time step
+        time_step_mech = data[pp.PARAMETERS][mech_kw]["time_step"]
+        np.testing.assert_equal(time_step_mech, np.array([1.0]))
+
+        # ----------> Check flow parameters
+        flow_kw = model.scalar_parameter_key
+        # Boundary condition type
+        bc_flow = data[pp.PARAMETERS][flow_kw]["bc"]
+        assert np.all(bc_flow.is_dir)
+        assert not np.all(bc_flow.is_neu)
+        assert not np.all(bc_flow.is_rob)
+        # Boundary condition values
+        bc_flow_vals = data[pp.PARAMETERS][flow_kw]["bc_values"]
+        np.testing.assert_equal(bc_flow_vals, np.array([0.0, 0.0, 0.0, 0.0]))
+        # Mass weights
+        mass_weight = data[pp.PARAMETERS][flow_kw]["mass_weight"]
+        np.testing.assert_equal(mass_weight, np.array([1.0]))
+        # Biot-alpha
+        biot_alpha_flow = data[pp.PARAMETERS][flow_kw]["biot_alpha"]
+        np.testing.assert_equal(biot_alpha_flow, np.array([1.0]))
+        # Source
+        source = data[pp.PARAMETERS][flow_kw]["source"]
+        np.testing.assert_equal(source, np.array([0.0]))
+        # Second order tensor
+        perm = data[pp.PARAMETERS][flow_kw]["second_order_tensor"].values
+        np.testing.assert_equal(perm, pp.SecondOrderTensor(np.array([1.0])).values)
+        # Vector source
+        vector_source = data[pp.PARAMETERS]["flow"]["vector_source"]
+        np.testing.assert_equal(vector_source, np.array([0.0, 0.0]))
+        # Ambient dimension
+        ambient_dim = data[pp.PARAMETERS]["flow"]["ambient_dimension"]
+        assert ambient_dim == 2
+
+    # ----------> Setting up parameters corresponding to the test of non-default parameters
+
+    # -----> Mixed-dimensional grid
+    mdg, box = pp.md_grids_2d.single_vertical(mesh_args=np.array([2, 1]), simplex=False)
+    sd_prim = mdg.subdomains(dim=2)[0]
+    data_prim = mdg.subdomain_data(sd_prim)
+    sd_sec = mdg.subdomains(dim=1)[0]
+    data_sec = mdg.subdomain_data(sd_sec)
+    intf = mdg.interfaces(dim=1)[0]
+    data_inf = mdg.interface_data(intf)
+
+    # -----> Time parameters
+    time_steps = [2.0, 5.0, 100.0]
+    end_times = [2.0, 10.0, 300.0]
+
+    # -----> Scalar bc types
+    all_bf = sd_prim.get_boundary_faces()
+    tol = 1e-10
+    east = sd_prim.face_centers[0] > box["xmax"] - tol
```

Comment:
These can be obtained by self._domain_boundary_sides

Reply:
Yes, but at this point, I don't have access to the model object yet. That's why I don't use the `self._domain_boundary_sides()` method.